### PR TITLE
Feature/user profile enhancements

### LINKED ISSUE #13: Make email not mandatory, or clearly state that it will be shown
More than one person request the ability to mask the email from being publicly shown

### LINKED ISSUE #15: About vuepeople.org button doesn't do anything
Should route back to the white left panel.

### LINKED ISSUE #22: Profile form: prettier error messages
Currently the messages are a bit basic

### LINKED ISSUE #26: User profile form: missing modified date

### PR DESCRIPTION
**Pull request by lordfuoco**
Several fixes / Enhancements:
fixes #13 
fixes #15 
fixes #22 
fixes #26

### CODE REVIEW

**Comment by lordfuoco** [comment]
@nullcode still missing the logic to exclude the email from the api answers in the appropriate Views. I did not wanted to mess with it since I am not sure which is the real proper way to do so since is possible to do it with at least 5 different methods.

**Comment by lordfuoco** [comment]
@nullcode small change on the email validator, I tested it and it works.
I think is done, waiting for your approval

**Comment by nullcode** [comment]
@lordfuoco looks, good. Please test this scenario manually: set your email to hidden, logout-login, change your email
If that works you can merge. Thanks

File: django/people/serializers.py [hunk 1]
```diff
@@ -20,8 +20,8 @@ def to_representation(self, obj):  # GET
             return ""
 
     def to_internal_value(self, data):  # POST
-        if data and not validate_email(data):
-            raise ValidationError("Wrong email format")
```
Comment (nullcode) [inline django/people/serializers.py]: remove ValidationError from imports

**Comment by lordfuoco** [comment]
@nullcode scenarios tested, all working, removed unused import. I'll merge on develop branc

**Pull request merged**